### PR TITLE
More transcript

### PR DESCRIPTION
**Pull request by mccalluc**
Rebased PR. @afred?

### CODE REVIEW

File: app/assets/stylesheets/locals/plain.css.scss [hunk 1]
```diff
@@ -10,5 +10,11 @@
             display: none;
             // Title at top of transcript isn't necessary: We know what we're looking at.
         }
+        .play-from-here::before {
+            content: '\25B6'; // right-pointing triangle
+            cursor: pointer;
+            color: $dark-blue;
+            padding-right: 0.3em;
+        }
```
Comment (afred) [inline app/assets/stylesheets/locals/plain.css.scss]: Why use `::before` pseudoclass?

Comment (mccalluc) [inline app/assets/stylesheets/locals/plain.css.scss]: @afred: I didn't want to put the arrow glyph in the actual html: If I copy the text from my screen, I don't want those arrows.

As best I understand, the CSS content only works on ::before or ::after.

Comment (afred) [inline app/assets/stylesheets/locals/plain.css.scss]: @mccalluc injecting content via CSS sets off a red flag for me. But I guess in this case we're using content (characters) as style (arrows), so it's a valid exception to the rule.

merging

**Comment by afred** [comment]
@mccalluc and without having tested it, I trust that clicking on the `::before` content of an element will trigger said element's `click()` event?

**Comment by mccalluc** [comment]
@afred: yeah, worked for me. I should see if this can be capybara tested: Have we tested the JS around an html5 player before?

**Comment by afred** [comment]
@mccalluc don't think so

**Pull request merged**